packages/esm-patient-tests-app/src/test-results/grouped-timeline/timeline-data-group.component.tsx

```diff
@@ -0,0 +1,186 @@
+import React, { useCallback, useContext, useEffect, useRef } from 'react';
```

Comment:
This whole file was just refactored out of grouped-timeline.component.tsx. No substantive changes.